pyproject.toml

```
@@ -31,6 +31,8 @@ dependencies = [
   "standard-distutils~=3.11.9; python_version>='3.11'",
   "databricks-bb-analyzer~=0.1.7",
   "sqlglot==26.1.3",
+  #"databricks-labs-blueprint[yaml]>=0.11.2,<0.12.0",
+  "databricks-labs-blueprint @ git+https://github.com/databrickslabs/blueprint@workaround-login-databricks-host",
```

Before merging this needs to be changed to:
```suggestion
  "databricks-labs-blueprint[yaml]>=0.11.2,<0.12.0",
```
(Assuming the upstream change is included in 0.11.2.)

---

Done.